example/spec/parallel/pythonReferenceTests/PythonReferenceSpec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const workflowName = 'PythonReferenceWorkflow';
```

Comment:
Just me: but I find this more readable if variable assignments come after require statements